docs/design.md

```diff
@@ -4,12 +4,12 @@ Just some random notes about the design of the sidecar injector
 
 ## Mixin Injections
 
-On the note of why the `k8s-sidecar-injector` does not support multiple injections (aka "mixin" injections), https://github.com/tumblr/k8s-sidecar-injector/issues/17 articulates my thought process during design fairly well. Copying the body of the issue for posterity:
+On the note of why the `k8s-sidecar-injector` does not support multiple injections (aka "mixin" injections), https://github.com/riege/k8s-sidecar-injector/issues/17 articulates my thought process during design fairly well. Copying the body of the issue for posterity:
```

Comment:
Broken link

Reply:
I'm deleting the design document, since this is not by us